src/routes/load/Load.test.tsx

```diff
@@ -53,17 +56,16 @@ describe('<Load>', () => {
       render(<Load />, customState)
 
       expect(mockedEndpoints.getSafeInfo).toBeCalledTimes(0)
-
-      const safeAddressInputNode = screen.getByTestId('load-safe-address-field')
-
-      fireEvent.change(safeAddressInputNode, { target: { value: validSafeAddress } })
-
-      expect(mockedEndpoints.getSafeInfo).toBeCalledWith(getClientGatewayUrl(), networkId, validSafeAddress)
-      // FIXME: should only call 1 time to the getSafeInfo endpoint see https://github.com/gnosis/safe-react/issues/2668
-      // expect(mockedEndpoints.getSafeInfo).toBeCalledTimes(1)
+      await waitFor(() => {
```

Comment:
Another instance where you should theoretically be using `act` (unless you are waiting for something to load).